_pages/home/updating-to-version-2211.35/modernize-apps-migrated-from-6.8-to-2211.19.md

```diff
@@ -1,466 +1,473 @@
+## Automatic Migration
 
+Spartacus includes specially-prepared schematics to automatically modernize your app to be as similar as possible to a newly created Angular 17 app.
 
-# Automatic migration
+**Note:** These schematics were only released in `@spartacus/schematics2211.36.0`, but you need to run them before fully upgrading your app to 2211.36. As a result, you need to install `@spartacus/schematics@2211.36.0` in a temporary directory and run the migration schematics from there.
 
-The Spartacus team provides special schematics that automatically modernize the app to look as much as possible like the new Angular 17 apps.
+1. Create a temporary sibling directory for the isolated 2211.36 installation schematics by running the following command from the root directory of your project:
 
-Note: the tool was released only in `@spartacus/schematics` v2211.35.0, but you need to run it before fully upgrading your app to this version.
-To do this, you'll need to install `@spartacus/schematics@2211.35.0` in a temporary directory and execute the migration schematic from there.
+   ```bash
+   node -e "require('fs').mkdirSync('../temp-schematics-36')"
+   ```
 
-Please run those commands from your project root directory:
+2. Install the schematics in the temporary directory by running the following command:
 
-```bash
-# 1. Create a temporary sibling directory for the isolated Schematics v2211.35 installation
-node -e "require('fs').mkdirSync('../temp-schematics-35')"
+   ```bash
+   npm install @spartacus/schematics@2211.36.0 --prefix ../temp-schematics-36
+   ```
 
-# 2. Install schematics in the temporary directory
-npm install @spartacus/schematics@2211.35.0 --prefix ../temp-schematics-35
+3. From the temporary directory, run the schematics on your project by running the following command:
 
-# 3. Execute in your project the schematics from the temporary directory
-ng g ../temp-schematics-35/node_modules/@spartacus/schematics:modernize-app-migrated-from-6_8-to-2211_19
+   ```bash
+   ng g ../temp-schematics-36/node_modules/@spartacus/schematics:modernize-app-migrated-from-6_8-to-2211_19
+   ```
 
-# 4. Clean up the temporary directory
-node -e "require('fs').rmSync('../temp-schematics-35', { recursive: true, force: true })"
-```
+4. Clean up the temporary directory by running the following command:
+
+   ```bash
+   node -e "require('fs').rmSync('../temp-schematics-36', { recursive: true, force: true })"
+   ```
 
-In case of any issues during the automatic migration, you can always fall back to the manual migration steps below.
+**Note:** For apps that use server-side rendering (SSR), now that you have completed this migration, you will need to run SSR and prerendering scripts in a different way. This is explained in detail in the section [New Commands for SSR Projects](#new-commands-for-ssr-projects), below.
```

Comment:
This is a significant change. And I'm afraid putting it in a separate section (that one can achieve only by jumping via a link) can lead to a high rate of customers skipping this important steps included there, therefore increasing unintentionally their friction on debugging the problem later.

Only wow I realised I'm in favor of **moving here directly** the contents of the section `New Commands for SSR Projects`, before we finally say "Congratulations!...". Would it work for you, Matt?

Reply:
Excellent point! I have moved the section up and made the necessary adjustments. Thank you!